controllers/articles.controller.js

```diff
@@ -31,6 +31,14 @@ const getCommentsByArticleId = (req,res,next) => {
 
 const postCommentsByArticleId = (req,res,next) => {
 
+    const requiredKeys = ['body', 'username']
+
+    for (const key in req.body) {
+        if (!requiredKeys.includes(key)) {
+            delete req.body[key]
+        }
+    }
+
```

Comment:
I don't think this step is necessary - we should just be able to access the properties we need with dot notation and ignore the other ones without having to actually delete them